src/components/organisms/Navbar/DesktopNavigation.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC } from "react";
+import * as NavigationMenuPrimitive from "@radix-ui/react-navigation-menu";
+import { CSS } from "@stitches/react/types/css-util";
+import {
+  DebugCard,
+  DesktopList,
+  DesktopListItem,
+  DesktopNavbarContainer,
+  DesktopViewport,
+  DropdownIndicator,
+  NavigationMenuContent,
+  NavigationMenuLink,
+  NavigationMenuList,
+  NavigationMenuTrigger,
+} from "./components/desktop";
+
+export interface DesktopProps {
+  children?: Element[] | Element | string;
+  title?: string;
+  href?: string;
+  css?: CSS<{}, {}, {}, {}>;
+}
+
+export const NavigationMenuItem = NavigationMenuPrimitive.Item;
+
+export const DesktopNavigation = () => (
+  <DesktopNavbarContainer>
+    <NavigationMenuList>
+      <NavigationMenuItem>
```

Comment:
Each menu item should generate from an object. This has a lot of repetitive code which can be simplified. Something like this perhaps?

```js
const tabs = [
  {
    name: "For Studenter",
    layout: "one",
    showOnMobile: false,
    Items: [{ title: "Webshop", link: "https://webshop.com", text: "Verdens best webshop"}, {...}]
  }
]
```